apps/topics-map/Dockerfile

```diff
@@ -5,7 +5,10 @@
 FROM gcr.io/mcback/common:latest
 
 # Install Java
-RUN apt-get -y --no-install-recommends install openjdk-8-jre-headless
+RUN \ 
+    apt-get -y update && \ 
```

Comment:
Another note - here's why one sometimes needs to run `apt-get update`:

https://github.com/mediacloud/backend/blob/master/doc/docker_issues.markdown#apt-get-no-longer-works-it-tells-me-to-maybe-run-apt-get-update

It's annoying, so let me know if you think of a good solution for this.

Reply:
maybe a cron to rebuild `base` once a week or something?

Reply:
A daily / weekly cron rebuild would work, the job would have to:

* Skip pulling `base` container image when pulling pre-built images to trigger a full rebuild;
* Doing the rest as usual.

If you're up for it, think it's worth it, and are ready to mess around with GitHub Actions a little bit more, you can do it, but that would have to be a separate issue in any case.

Reply:
Made an issue [here](https://github.com/mediacloud/backend/issues/775) but seems like something to put on the backburner for now while I work on my other tasks.

Reply:
Agree on the backburner note.